tracer/src/Datadog.Trace.Bundle/Datadog.Trace.Bundle.targets

```diff
@@ -0,0 +1,88 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="RemoveDatadogNuGetFiles" AfterTargets="Build;Publish" Condition="'$(RemoveUnusedDatadogFiles)' == 'true'">
+    <PropertyGroup>
+      <IsPublish Condition="'$(PublishDir)' != ''">true</IsPublish>
```

Comment:
TODO: fix this, `PublishDir` is always defined so this doesn't work as I expected

Reply:
refactored to run after `ComputeResolvedFilesToPublishList`